### DISCUSSION TITLE
美国EB绿卡名额及排期计算相关Wiki

### DISCUSSION THREAD

**Discussion by GentlemanXu** [The-Run-Philosophy-Organization/run#148] [General]
[https://ebgreencard.com](https://ebgreencard.com) 上有比较详细的关于绿卡排期与名额分配的科普，可以考虑引用

**Comment by XiGou** [comment]
这么来看 ,EB 卡难度大于考上清华啊

**Comment by GentlemanXu** [comment]
最近两年大陆出生申请人EB2的三年排期是因为
1. 吃到了亲属移民(FB)的没用掉名额(Spill Over)，导致总名额大致翻倍，中国申请人得到2x7%配额。2022年10月开始中国大陆申请人名额将恢复正常。
2. 许多人在之前EB3排期领先的时候从EB2降级到EB3，导致EB2排期大进，领先于EB2。如果先前降级至EB3的申请人重新升级会EB2将导致EB2排期停滞甚至倒退，

未来10年内如果没有移民改革去除7%国别限制或者增加总名额，2022年提交I-140的大陆EB2申请人预计将面临5年-15年的排期
